packages/react/src/IDE/IDE.tsx

```diff
@@ -129,17 +129,25 @@ type IDEChatProps = {
   'data-testid'?: string
 } & BaseProps<HTMLElement>
 
-type MessageRole = 'user' | 'assistant'
-
-export type IDEChatMessage = {
-  role: MessageRole
+type IDEChatMessageUser = {
+  role: 'user'
   handle: string
   avatar: string
   message: string
   codeSnippet?: string
-  highlighter?: 'hljs' // add additional highlighters as needed
+  highlighter?: 'hljs'
+}
+
+type IDEChatMessageAssistant = {
+  role: 'assistant'
+  handle: string
+  message: string
+  codeSnippet?: string
+  highlighter?: 'hljs'
 }
 
+export type IDEChatMessage = IDEChatMessageUser | IDEChatMessageAssistant
```

Comment:
We'll need a changeset for this update as it's a material change to the library. Although, it doesn't need to say too much. 

Maybe something like: Updated `IDEChatMessage` type declaration to differentiate between assistant and user messages. WDYT?

Reply:
Good shout, added 👍